clouddriver-kubernetes/src/main/java/com/netflix/spinnaker/clouddriver/kubernetes/op/handler/KubernetesDeploymentHandler.java

```diff
@@ -82,7 +86,7 @@ public KubernetesKind kind() {
 
   @Override
   public boolean versioned() {
-    return false;
+    return true;
```

Comment:
This change is not OK.  It's not a matter of fixing up tests...this will break all kind of expectations that real-world users have.

Reply:
I will close this Draft PR due to the escalation of the issue